test/ui/test_spec.js

```diff
@@ -168,6 +168,37 @@ describe('UI tests', function() {
         .then(done);
     });
 
+    it('Settings page defaults to light theme', function(done) {
+      driver.findElement(By.id('accountDropdownButton')).click()
+        .then(() => screenshotAndCompare('settingsMenu.png', 'settingsMenu', null))
+        .then(() => driver.findElement(By.id('settingsButton')).click())
+        // After clicking on the Settings menu item, wait for the dialog.
+        .then(() => driver.wait(until.elementLocated(By.id('lightThemeRadioOption'))))
+        // Wait for the fade-in to complete.
+        .then(() => driver.sleep(1000))
```

Comment:
We need some kind of delay to wait for the fade-in to complete. There is probably a way to do it with a driver.wait and some condition, but after I spent ten minutes playing with it I decided this was good enough. If you happen to know of an event or something that we can catch when the fade-in completes, that could be a reasonable way to deal with it. Or maybe we could disable the fade-in.